client/app/auth/auth.js

```diff
@@ -1,4 +1,48 @@
-angular.module('courageousTrapeze.auth', [])
+var app = angular.module('courageousTrapeze.auth', []);
```

Comment:
@mwharris333 Let's avoid using the variable name, `app`, for this auth module. The reason being, `app` is usually for the main module (defined in app.js). If you want to use a variable name for this auth module, perhaps a better variable name could be `auth` or `authModule`.

Another approach is to not use a variable and instead chain the controller like so:

```
angular.module('courageousTrapeze.auth', [])
.controller('AuthController', ['$scope', 'Auth', function($scope, Auth) {
  ...
}]);
```